worlds/pokemon_rb/__init__.py

```diff
@@ -67,7 +67,16 @@ class PokemonWebWorld(WebWorld):
         ["Shiny"]
     )
 
-    tutorials = [setup_en, setup_es]
+    setup_es = Tutorial(
```

Comment:
This line should be `setup_fr = Tutorial(`, but otherwise looks good (at least for this part, I can't read French to review the documentation itself).